### PR TITLE
scala 2.12.7

### PR DESCRIPTION
**Pull request by bjaglin**

### CODE REVIEW

**Comment by bjaglin** [comment]
```
[error] 	* org.scala-lang.modules:scala-xml_2.12:2.1.0 (early-semver) is selected over 1.2.0
[error] 	    +- org.scala-lang:scala-compiler:2.12.17              (depends on 2.1.0)
[error] 	    +- org.scalatest:scalatest_2.12:3.0.8                 (depends on 1.2.0)
```
https://github.com/scala/bug/issues/12632
I will ignore that

**Comment by tgodzik** [comment]
It's the same issue as https://github.com/scalameta/scalameta/pull/2861#issuecomment-1247875096

**Comment by bjaglin** [comment]
> It's the same issue as [scalameta/scalameta#2861 (comment)](https://github.com/scalameta/scalameta/pull/2861#issuecomment-1247875096)

I bet this will be a trending google search query in the coming days :)

File: project/ScalafixBuild.scala [hunk 1]
```diff
@@ -193,7 +193,9 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
     // coursier-versions always return false for the *.*.*.*-r pattern jgit uses
     libraryDependencySchemes += Dependencies.jgit.withRevision("always"),
     // silence warning for 2.7.0 -> 3.0.0
-    libraryDependencySchemes += "com.lihaoyi" %% "sourcecode" % "always"
+    libraryDependencySchemes += "com.lihaoyi" %% "sourcecode" % "always",
+    // https://github.com/scala/bug/issues/12632
+    libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml" % "always"
```
Comment (bjaglin) [inline project/ScalafixBuild.scala:198]: https://github.com/scalacenter/scalafix/pull/1671

File: project/ScalafixBuild.scala [hunk 2]
```diff
@@ -193,11 +186,13 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
...
   )
 
   override def projectSettings: Seq[Def.Setting[_]] = List(
-    versionPolicyIntention := versionPolicyIntentionSetting.value,
+    versionPolicyIntention := Compatibility.None,
```
Comment (bjaglin) [inline project/ScalafixBuild.scala:195]: https://github.com/scalacenter/scalafix/commit/2e2691cb9b43d32f59e8b19b87d19385ff0a40af

**Pull request merged**